tests/cypress/template/vocab-home.cy.js

```diff
@@ -61,7 +61,7 @@ describe('Vocabulary home page', () => {
     cy.get('#tab-alphabetical').contains('a', 'care institutions').click()
 
     // check the concept prefLabel
-    cy.get('#concept-heading h1').invoke('text').should('equal', 'care institutions')
+    cy.get('#concept-heading h1', {'timeout': 15000}).invoke('text').should('equal', 'care institutions')
```

Comment:
Is this change necessary?

Reply:
It was failing before without the timeout but now it isn't anymore. I'll take it out